.travis.yml

```diff
@@ -40,6 +40,10 @@ matrix:
       env:
         VERSION="11.0" ODOO_REPO="OCA/OCB" TESTS="1" LINT_CHECK="0"
         INCLUDE="test_module,second_module"
+    - python: 3.7-dev
```

Comment:
Why dev?

Reply:
The list of the documentation has dev.
I think that is because they don’t have updated the image base with the recently released stable yet.

https://docs.travis-ci.com/user/languages/python/#specifying-python-versions

Reply:
I guess those are just examples. Have you actually tested it without the -dev?